tree/tree/src/InternalTreeUtils.cxx

```diff
@@ -268,7 +276,8 @@ ROOT::TreeUtils::RFriendInfo GetFriendInfo(const TTree &tree, bool retrieveEntri
    }
 
    return ROOT::TreeUtils::RFriendInfo{std::move(friendNames), std::move(friendFileNames),
-                                       std::move(friendChainSubNames), std::move(nEntriesPerTreePerFriend)};
+                                       std::move(friendChainSubNames), std::move(nEntriesPerTreePerFriend),
+                                       std::move(treeIndexes)};
```

Comment:
`treeIndexes` is a pointer.  What is the point being made here by using `std::move`?

Reply:
`treeIndexes` is a `std::vector<TreeIndexInfos>`

Reply:
I see. I got confused by `treeIndex` vs `treeIndexes` while scanning the diff.